apps/cowswap-frontend/src/modules/bridge/pure/contents/QuoteBridgeContent/index.tsx

```
@@ -0,0 +1,78 @@
+        <ReceiveAmountTitle>
+          <b>Min. to receive</b>
+        </ReceiveAmountTitle>
+      ),
+      content: children ? buyAmountEl : <b>{buyAmountEl}</b>,
+    },
+  ]
+
+  return (
+    <>
+      {contents.filter(isTruthy).map(({ withTimelineDot, label, content }) => (
+        <ConfirmDetailsItem withTimelineDot={withTimelineDot} label={label}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)